From: Remi Collet <remi@remirepo.net>
Date: Tue, 7 Jun 2022 09:57:15 +0200
Subject: NEWS

(cherry picked from commit f451082baf14ee9ea86cdd19870e906adb368f02)
(cherry picked from commit 87247fb08e905e629836350ac4e639edd1b40ed8)
(cherry picked from commit 151499ec0f70bf4f1bd65aebf037bd6273f0ef34)
(cherry picked from commit b243ab09f95d2737b99ce87d485c052734c2f3f5)
(cherry picked from commit 75c7daa8593606e3ce0f6247f9ed1027257d262f)
---
 NEWS | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/NEWS b/NEWS
index 34d463e..e81d259 100644
--- a/NEWS
+++ b/NEWS
@@ -1,6 +1,16 @@
 PHP                                                                        NEWS
 |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
 
+Backported from 7.4.30
+
+- mysqlnd:
+  . Fixed bug #81719: mysqlnd/pdo password buffer overflow.
+    (CVE-2022-31626) (c dot fol at ambionics dot io)
+
+- pgsql
+  . Fixed bug #81720: Uninitialized array in pg_query_params().
+    (CVE-2022-31625) (cmb)
+
 Backported from 7.3.33
 
 - XML:
